account_move_helper/models/res_partner.py

```diff
@@ -57,6 +57,8 @@ def _compute_new_debit_credit(self):
 
```

Comment:
**New debit** and **New credit** fields: remove string (because is same as field name)